lib/ecto/query.ex

```diff
@@ -475,11 +475,34 @@ defmodule Ecto.Query do
   @doc """
   Resets a previously set field on a query.
 
-  It can reset any query field except the query source (`from`).
+  It can reset many fields except the query source (`from`).
 
   ## Example
```

Comment:
Can you use the opportunity to rename this to "## Examples"? Tks!

Reply:
updated